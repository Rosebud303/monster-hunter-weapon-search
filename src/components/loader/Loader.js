import React from 'react';


const Loader = () => {
  return (
    <div>
      <h1>Is Loading...</h1>
      <img src={'https://media.giphy.com/media/zocX5Vxv9Bs9q/giphy.gif'} alt='Is Loading' />
    </div>
  )
}

export default Loader;