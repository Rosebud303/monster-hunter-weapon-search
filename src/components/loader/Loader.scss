.loader {
  position: absolute;
  width: 100%;
  text-align: center;
  color: white;
  background: rgba(0,0,0,.5);
}

.load-img {
  margin-bottom: 25px;
}