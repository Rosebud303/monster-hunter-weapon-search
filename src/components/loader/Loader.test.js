import React from 'react';
import { shallow } from 'enzyme';
import Loader from './Loader'


describe('', () => {
  let wrapper;  
  beforeEach(() => {
    wrapper = shallow (
      <Loader />
    )
  });

  it('should match snap shot', () => {
    expect(wrapper).toMatchSnapshot()
  });

});