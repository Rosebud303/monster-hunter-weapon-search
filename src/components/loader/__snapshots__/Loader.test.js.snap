// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snap shot 1`] = `
<div
  className="loader"
>
  <h1>
    Loading...
  </h1>
  <img
    alt="Is Loading"
    className="load-img"
    src="https://media.giphy.com/media/zocX5Vxv9Bs9q/giphy.gif"
  />
</div>
`;
